scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -785,6 +795,9 @@ private void processAlertRecord(final Record record, final Connection connection
 					errorHandler.onError(alert.getPeer(), alert.getLevel(), alert.getDescription());
```

Comment:
When I made this kind of feedback, it doesn't mean that it should be done "immediately in this PR by this PR authors". It's mainly to know on what we agree.
I will open an issue and we can do that later.